### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by tethryus**
Included Test Coverage
Included Code Climate
Signed off by Sorin Moldovan: <sorin.moldovan@hpe.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,46 @@
...
+### Open Source Project
+
+[CloudSlang Project website](http://cloudslang.io/#/)
+
+[CloudSlang Content source code](https://github.com/CloudSlang/cloud-slang-content)
+
+[CloudSlang Documentation](http://cloudslang-docs.readthedocs.io/en/v1.0/)
+
+
+## Get Involved!!!
```
Comment (Bonczidai) [inline README.md]: mayeb link to our contribution guide from `cloudslang` repo

File: README.md [hunk 2]
```diff
@@ -1,6 +1,46 @@
...
+    cs-vmware - Integration with VMWare vCenter Server
+    cs-xml - XML parsing and manipulation actions
+
+### Open Source Project
+
+[CloudSlang Project website](http://cloudslang.io/#/)
+
+[CloudSlang Content source code](https://github.com/CloudSlang/cloud-slang-content)
+
+[CloudSlang Documentation](http://cloudslang-docs.readthedocs.io/en/v1.0/)
```
Comment (Bonczidai) [inline README.md]: how about latest here?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,46 @@
...
+* [Overview](#Overview)
+* [Common Integrations](#Common Integrations)
+* [Open Source Project](#Open Source Project)
+
+<a name="Overview"/>
+### Overview
+
+###### This repository contains the sources (java actions) to be used by operations located inside the cloud-slang-content repository
+
+Overall Permissions: **All**
```
Comment (Bonczidai) [inline README.md]: ?

File: README.md [hunk 4]
```diff
@@ -1,6 +1,46 @@
-cs-actions
+# cs-actions
 =============
 
-Java actions to be used by operations.
-
 [![Build Status](https://travis-ci.org/CloudSlang/cs-actions.svg?branch=master)](https://travis-ci.org/CloudSlang/cs-actions)
+[![Coverage Status](https://coveralls.io/repos/github/CloudSlang/cs-actions/badge.svg?branch=master)](https://coveralls.io/github/CloudSlang/cs-actions?branch=master)
+[![Code Climate](https://codeclimate.com/github/CloudSlang/cs-actions/badges/gpa.svg)](https://codeclimate.com/github/CloudSlang/cs-actions)
```
Comment (Bonczidai) [inline README.md]: does the badge show up for you correctly?

File: README.md [hunk 5]
```diff
@@ -1,6 +1,46 @@
-cs-actions
+# cs-actions
 =============
 
-Java actions to be used by operations.
-
 [![Build Status](https://travis-ci.org/CloudSlang/cs-actions.svg?branch=master)](https://travis-ci.org/CloudSlang/cs-actions)
+[![Coverage Status](https://coveralls.io/repos/github/CloudSlang/cs-actions/badge.svg?branch=master)](https://coveralls.io/github/CloudSlang/cs-actions?branch=master)
```
Comment (Bonczidai) [inline README.md]: from coveralls link:
```
BADGE YOUR REPO: CS-ACTIONS
We detected this repo isn’t badged! Grab the embed code to the right, add it to your repo to show off your code coverage, and when the badge is live hit the refresh button to remove this message.
```

Comment (tethryus) [inline README.md]: The badge has to be on master in order for it to show correctly

File: README.md (hunk 3 above)
Comment (tethryus) [inline README.md]: Removed

Comment (tethryus) [inline README.md]: Removed

**Comment by tethryus** [comment]
@bonczidai  please review

**Comment by Bonczidai** [comment]
@tethryus please check the badges

**Comment by tethryus** [comment]
At least one of the badges will work as soon as it will be merged

**Pull request merged**